[2020-11-13 08:22:29,217][ERROR][2403655][experiments.py:run_experiment_from_to_file:752] XXX failed 14 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 08:22:29,217][ERROR][2403655][experiments.py:run_experiment_from_to_file:752] XXX failed 14 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 08:33:08,784][ERROR][2411678][experiments.py:run_experiment_from_to_file:752] XXX failed 64 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 08:33:08,784][ERROR][2411678][experiments.py:run_experiment_from_to_file:752] XXX failed 64 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 08:33:11,960][ERROR][2411747][experiments.py:run_experiment_from_to_file:752] XXX failed 66 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'NoneType'
[2020-11-13 08:33:11,960][ERROR][2411747][experiments.py:run_experiment_from_to_file:752] XXX failed 66 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'NoneType'
[2020-11-13 08:46:09,346][ERROR][2424122][experiments.py:run_experiment_from_to_file:752] XXX failed 114 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 08:46:09,346][ERROR][2424122][experiments.py:run_experiment_from_to_file:752] XXX failed 114 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 08:48:03,562][ERROR][2425656][experiments.py:run_experiment_from_to_file:752] XXX failed 120 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'NoneType'
[2020-11-13 08:48:03,562][ERROR][2425656][experiments.py:run_experiment_from_to_file:752] XXX failed 120 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'NoneType'
[2020-11-13 09:24:25,952][ERROR][2452775][experiments.py:run_experiment_from_to_file:752] XXX failed 223 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data agent 3 with malfunction ExperimentMalfunction(time_step=1361, agent_id=3, malfunction_duration=50). Found earliest=1362 for Waypoint(position=(64, 62), direction=2)
[2020-11-13 09:24:25,952][ERROR][2452775][experiments.py:run_experiment_from_to_file:752] XXX failed 223 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data agent 3 with malfunction ExperimentMalfunction(time_step=1361, agent_id=3, malfunction_duration=50). Found earliest=1362 for Waypoint(position=(64, 62), direction=2)
[2020-11-13 09:57:44,548][ERROR][2469678][experiments.py:run_experiment_from_to_file:752] XXX failed 293 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data agent 3 with malfunction ExperimentMalfunction(time_step=628, agent_id=3, malfunction_duration=50). Found earliest=629 for Waypoint(position=(64, 62), direction=2)
[2020-11-13 09:57:44,548][ERROR][2469678][experiments.py:run_experiment_from_to_file:752] XXX failed 293 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data agent 3 with malfunction ExperimentMalfunction(time_step=628, agent_id=3, malfunction_duration=50). Found earliest=629 for Waypoint(position=(64, 62), direction=2)
[2020-11-13 10:14:42,151][ERROR][2474579][experiments.py:run_experiment_from_to_file:752] XXX failed 322 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'float'
[2020-11-13 10:14:42,151][ERROR][2474579][experiments.py:run_experiment_from_to_file:752] XXX failed 322 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'float'
[2020-11-13 10:18:53,381][ERROR][2476443][experiments.py:run_experiment_from_to_file:752] XXX failed 329 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'float'
[2020-11-13 10:18:53,381][ERROR][2476443][experiments.py:run_experiment_from_to_file:752] XXX failed 329 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'float'
[2020-11-13 10:19:59,464][ERROR][2476897][experiments.py:run_experiment_from_to_file:752] XXX failed 331 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 10:19:59,464][ERROR][2476897][experiments.py:run_experiment_from_to_file:752] XXX failed 331 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 10:22:15,177][ERROR][2477712][experiments.py:run_experiment_from_to_file:752] XXX failed 338 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 10:22:15,177][ERROR][2477712][experiments.py:run_experiment_from_to_file:752] XXX failed 338 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 11:04:04,780][ERROR][2493355][experiments.py:run_experiment_from_to_file:752] XXX failed 413 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'float'
[2020-11-13 11:04:04,780][ERROR][2493355][experiments.py:run_experiment_from_to_file:752] XXX failed 413 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'float'
[2020-11-13 11:58:25,521][ERROR][2513571][experiments.py:run_experiment_from_to_file:752] XXX failed 485 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 11:58:25,521][ERROR][2513571][experiments.py:run_experiment_from_to_file:752] XXX failed 485 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'float' and 'NoneType'
[2020-11-13 13:30:20,638][ERROR][2542299][experiments.py:run_experiment_from_to_file:752] XXX failed 571 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 13:30:20,638][ERROR][2542299][experiments.py:run_experiment_from_to_file:752] XXX failed 571 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 14:12:01,746][ERROR][2553171][experiments.py:run_experiment_from_to_file:752] XXX failed 608 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 14:12:01,746][ERROR][2553171][experiments.py:run_experiment_from_to_file:752] XXX failed 608 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 14:19:13,516][ERROR][2535532][experiments.py:run_experiment_from_to_file:752] XXX failed 554 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data 
[2020-11-13 14:19:13,516][ERROR][2535532][experiments.py:run_experiment_from_to_file:752] XXX failed 554 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data 
[2020-11-13 16:00:10,676][ERROR][2582660][experiments.py:run_experiment_from_to_file:752] XXX failed 714 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'NoneType'
[2020-11-13 16:00:10,676][ERROR][2582660][experiments.py:run_experiment_from_to_file:752] XXX failed 714 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data unsupported operand type(s) for +: 'NoneType' and 'NoneType'
[2020-11-13 16:10:05,252][ERROR][2584484][experiments.py:run_experiment_from_to_file:752] XXX failed 726 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 16:10:05,252][ERROR][2584484][experiments.py:run_experiment_from_to_file:752] XXX failed 726 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 16:32:10,489][ERROR][2588978][experiments.py:run_experiment_from_to_file:752] XXX failed 743 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 16:32:10,489][ERROR][2588978][experiments.py:run_experiment_from_to_file:752] XXX failed 743 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 16:48:35,936][ERROR][2587530][experiments.py:run_experiment_from_to_file:752] XXX failed 734 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 16:48:35,936][ERROR][2587530][experiments.py:run_experiment_from_to_file:752] XXX failed 734 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=True
[2020-11-13 16:52:57,963][ERROR][2588954][experiments.py:run_experiment_from_to_file:752] XXX failed 742 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=False
[2020-11-13 16:52:57,963][ERROR][2588954][experiments.py:run_experiment_from_to_file:752] XXX failed 742 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data ASP solver: No solution found. Interrupted=False
[2020-11-13 17:18:07,549][ERROR][2602985][experiments.py:run_experiment_from_to_file:752] XXX failed 778 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data agent 78 with malfunction ExperimentMalfunction(time_step=969, agent_id=78, malfunction_duration=50). Found earliest=970 for Waypoint(position=(82, 65), direction=2)
[2020-11-13 17:18:07,549][ERROR][2602985][experiments.py:run_experiment_from_to_file:752] XXX failed 778 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data agent 78 with malfunction ExperimentMalfunction(time_step=969, agent_id=78, malfunction_duration=50). Found earliest=970 for Waypoint(position=(82, 65), direction=2)
[2020-11-13 17:22:02,417][ERROR][2587579][experiments.py:run_experiment_from_to_file:752] XXX failed 735 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data 
[2020-11-13 17:22:02,417][ERROR][2587579][experiments.py:run_experiment_from_to_file:752] XXX failed 735 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data 
[2020-11-13 17:23:06,716][ERROR][2587155][experiments.py:run_experiment_from_to_file:752] XXX failed 730 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data 
[2020-11-13 17:23:06,716][ERROR][2587155][experiments.py:run_experiment_from_to_file:752] XXX failed 730 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data 
[2020-11-13 17:29:19,989][ERROR][2602868][experiments.py:run_experiment_from_to_file:752] XXX failed 777 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data [Errno 28] No space left on device
[2020-11-13 17:29:19,989][ERROR][2602868][experiments.py:run_experiment_from_to_file:752] XXX failed 777 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data [Errno 28] No space left on device
[2020-11-13 17:30:34,829][ERROR][2603040][experiments.py:run_experiment_from_to_file:752] XXX failed 779 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data [Errno 28] No space left on device
[2020-11-13 17:30:34,829][ERROR][2603040][experiments.py:run_experiment_from_to_file:752] XXX failed 779 DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV/DENSITY_VARIATION_EXPERIMENTS_HUGE_ENV_baseline_2020_11_13T08_19_41/data [Errno 28] No space left on device
